graphkb/constants.py

```diff
@@ -172,3 +172,45 @@ def __getitem__(self, key):
     "mutations": "mut",
     "nonsense": ">",
 }
+
+# For match.type_screening() [KBDEV-1056]
+DEFAULT_NON_STRUCTURAL_VARIANT_TYPE = 'mutation'
+STRUCTURAL_VARIANT_SIZE_THRESHOLD = 50  # bp
+STRUCTURAL_VARIANT_TYPES = [
+    "structural variant",
+    "insertion",
+    "in-frame insertion",
+    "deletion",
+    "deletion polymorphism",
+    "in-frame deletion",
+    "translocation",
+    "inverted translocation",
+    "inversion",
+    "indel",
+    "fusion",
+    "out-of-frame fusion",
+    "oncogenic fusion",
+    "in-frame fusion",
+    "disruptive fusion",
+    "duplication",
+    "internal duplication",
+    "tandem duplication",
+    "internal tandem duplication",
+    "itd",
```

Comment:
Pretty sure 'itd' and 'internal tandem duplication' are the same thing.

Reply:
Yes they are, the former being deprecated by the later, but I kept both since both are still on the term tree:
![image](https://github.com/bcgsc/pori_graphkb_python/assets/4022327/4de3fd49-bc87-4efb-9c24-fa2912c46453)
These two shouldn't end up as variant notation to be matched anyway though.